src/App/__snapshots__/app.test.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App tests should render default App component 1`] = `
+<div
+  className="app-container"
+>
+  <div
+    className="notification-container"
+  >
+    <Alert
+      action={
+        <Button
+          aria-label={null}
+          className="dismiss-notification"
+          component="button"
+          isActive={false}
+          isBlock={false}
+          isDisabled={false}
+          isFocus={false}
+          isHover={false}
+          onClick={[Function]}
+          type="button"
+          variant="secondary"
+        >
+          Dismiss
+        </Button>
+      }
+      className=""
+      title="Setup Complete"
+      variant="success"
```

Comment:
When running this locally I see a new entry for `variantLabel={null}` that doesn't exist here so the snapshot test fails for me.

<img width="721" alt="screen shot 2019-02-06 at 2 29 45 pm" src="https://user-images.githubusercontent.com/5942899/52368162-c939c200-2a1b-11e9-9fc7-b962165a03bf.png">

Reply:
have you pulled the latest? I cant reproduce for some reason.